### PR TITLE
Content nsfw fixes

### PR DESCRIPTION
**Pull request by curioussavage**
:eyeglasses: @ajacksified

### CODE REVIEW

File: assets/js/client.es6.js [hunk 1]
```diff
@@ -239,13 +241,13 @@ function initialize(bindLinks) {
...
 
-  if (window.bootstrap.dataCache.user) {
-    app.setState('user', window.bootstrap.dataCache.user);
-    app.setState('preferences', window.bootstrap.dataCache.preferences);
+  if (dataCache.user) {
+    app.setState('user', dataCache.user);
+    app.setState('preferences', dataCache.preferences);
 
-    cookies.set('over18', window.bootstrap.dataCache.preferences.over_18);
+    cookies.set('over18', dataCache.preferences.body.over_18);
```
Comment (curioussavage) [inline assets/js/client.es6.js]: this was one of the big issues here. the preferences are nested in body.

File: assets/js/client.es6.js [hunk 2]
```diff
@@ -51,6 +51,7 @@ var $body = document.body || document.getElementsByTagName('body')[0];
 var $head = document.head || document.getElementsByTagName('head')[0];
 
 var config = defaultConfig();
+const dataCache = window.bootstrap.dataCache;
```
Comment (ajacksified) [inline assets/js/client.es6.js]: nuke here, since you defined `dataCache` within `initialize`

**Comment by ajacksified** [comment]
:nail_care:

**Comment by ajacksified** [comment]
Also, it doesn't show NSFW images (good) but it _also_ doesn't show the blurred pixel version (bad); it only shows the blue placeholder.

**Comment by curioussavage** [comment]
do you have an example? I am looking around for a non nsfw sub with nsfw images where there is a preview and I cant find one :(

**Comment by curioussavage** [comment]
Ok, so it will show the blurry images but only if 'make safer for work' is disabled. otherwise those images are not returned in the api response. (maybe the api should return that?)

:haircut:

**Comment by ajacksified** [comment]
:fish:

**Pull request closed without merging**